### PR TITLE
fixes #28

### LINKED ISSUE #28: Trailing comma in strfmt arg list will produce hard to track compiler error
Hey, 
first, let me say: thanks for this library, it is really handy for code generation :)

I found a problem that might be considered a bug. A trailing comma in the argument list of strfmt will produce a compiler error. Normal rust functions allow a trailing comma, so it might be a good idea to allow this here too. But if you don't want to allow it, it would be nice if the error would show up where the trailing comma is missing, instead of in macros.rs

Example:
```rust
use strfmt::strfmt;
fn main() {
    let working = strfmt!("This {verb}", verb => "works").unwrap();
    let not_working = strfmt!("This {verb}", verb => "produces a compiler error",).unwrap();
}
```
compiling this will produce the following error:

```
felix@TP-Felix:/tmp/strfmt_err_demo$ cargo run
   Compiling strfmt_err_demo v0.1.0 (/tmp/strfmt_err_demo)
error: unexpected end of macro invocation
  --> /home/felix/.cargo/registry/src/github.com-1ecc6299db9ec823/strfmt-0.2.2/src/macros.rs:48:31
   |
48 |         strfmt_builder!($vars,$($values)*)
   |                               ^ missing tokens in macro arguments

error: could not compile `strfmt_err_demo` due to previous error
```

### PR DESCRIPTION
**Pull request by KnorrFG**

### CODE REVIEW

File: src/tests/strfmt.rs [hunk 1]
```diff
@@ -191,6 +191,13 @@ fn test_ignore_missing() {
     run_tests(&values, &vars, &strfmt_ignore);
 }
 
+#[test]
+fn test_trailing_comma() {
```
Comment (vitiral) [inline src/tests/strfmt.rs:195]: just to be absolutely sure, please also assert the output is as-expected (it equals bar)

**Review by vitiral — APPROVED** [review]
Thanks, one minor nit -- otherwise looks good!

**Comment by vitiral** [comment]
Merged in c1e9af4

**Comment by vitiral** [comment]
published in version `0.2.3`

**Comment by KnorrFG** [comment]
Thanks :)

**Pull request closed without merging**